crates/neon-runtime/src/napi/primitive.rs

```diff
@@ -43,3 +43,9 @@ pub unsafe fn number_value(env: Env, p: Local) -> f64 {
     );
     value
 }
+
+/// Mutates the `out` argument provided to refer to a newly created `Local` containing a
+/// JavaScript symbol.
+pub unsafe fn symbol(out: &mut Local, env: Env, desc: Local) {
```

Comment:
Please remove the out pointer. That pattern is only used for exiting methods that require compatibility between the two backends. Since this is a new API, it can follow the new pattern of returning the type directly.